common/app/redux/index.js

```diff
@@ -66,7 +68,7 @@ const throwIfUndefined = () => {
 //   label?: String,
 //   value?: Number
 // }) => () => Object
-export const createEventMetaCreator = ({
+export function createEventMetaCreator({
```

Comment:
Why is this change made?

Reply:
I keep getting a weird `not a function` error, when I logged `typeof createEventMetaCreator`, I got `undefined`

Changing it to a function declaration fixed that.